### DISCUSSION TITLE
How to avoid deep sleep?

### DISCUSSION THREAD

**Discussion by RobinVanCauter** [seaky/nspanel_pro_tools_apk#85] [Q&A]
Whenever the device loses power, all goes well, the app starts, home assistant boots. But after a while of inactivity the deep sleep kicks in and the device is just a black unresponsive screen. I can awaken it with ADB over the network (power button simulation).

Is there any way to avoid the deep sleep to happen?

**Comment by seaky** [comment]
Hi

Turn on "resume on boot" it will do the trick.
